git-extra/git-askyesno.c

```diff
@@ -0,0 +1,23 @@
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+
+int wmain(int argc, wchar_t **wargv)
+{
+	int res;
+	wchar_t *title = L"Question?";
+
+	if (argc > 2 && !wcscmp(L"--title", wargv[1])) {
+		title = wargv[2];
+		argc -=2;
+		wargv += 2;
+	}
+
+	if (argc != 2) {
+		MessageBoxW(NULL, L"Required: exactly one argument", L"Error!", MB_OK);
```

Comment:
`git gui--askyesno` wrote it's error message to `stderr` and had a different wording, but this should work, too. Could be confusing for end users, though, if they get a messagebox with an obscure error due to a script calling this in a bogus way.

Reply:
Yeah, I never liked that behavior. The cases where `git askyesno` is to be called are when we do *not* have a console to interact with the user, after all. So `git gui--askyesno` would have written the message to essentially `/dev/null` and returned `1` as if the user had said "No". But the user did not even see a flicker.

Reply:
Maybe the proper approach would be a third return code for errors. Whoever calls `git gui--askyesno` should be checking the result in some way, but I guess that this might break callers that check if the result is !=1 or maybe do some conversion to a boolean value with it that doesn't expect a third value. Odd initial design chioce to use the same return value for errors and no.